src/platform/Zephyr/BLEManagerImpl.cpp

```diff
@@ -802,6 +802,20 @@ bool BLEManagerImpl::SendWriteRequest(BLE_CONNECTION_OBJECT conId, const ChipBle
     return false;
 }
 
+bool BLEManagerImpl::SendReadRequest(BLE_CONNECTION_OBJECT conId, const ChipBleUUID * svcId, const ChipBleUUID * charId,
```

Comment:
Why is this needed? Wasn't it recently deleted as unused?

Reply:
I thought it was unused but see it implemented somewhere else, can remember. I can remove it